### DISCUSSION TITLE
Stacked bar chart

### DISCUSSION THREAD

**Discussion by lauraocana** [Telefonica/mistica-design#1074] [Proposals]
Hola! Tenemos varios casos de uso en producción en Novum representados con una stacked bar chart horizontal pero este componente no está en Mistica por el momento. ¿Cabría la posibilidad de hacer core este componente o por el contrario sería preferible desarrollarlo como Mistica extended?
¡Muchas gracias!

<img width="560" alt="image" src="https://user-images.githubusercontent.com/68842873/220932480-ab2868ec-7f8d-4221-a853-12bfad7987a0.png">

**Comment by yceballost** [comment]
Buenas!

Hemos estado hablando en el equipo de cómo encajar el mundo gráficas dentro de Mística. Nos gustaría pensar cómo es la mejor forma de integrar gráficas en un DS de cara a otras necesidades de otros productos, para esto tenemos que hacer un análisis de otros productos donde se estén usando gráficas y encontrar un punto en el que el componente de Mística pueda cubrir la mayoría de necesidades.

Nos inclinamos por tener un enfoque más atómico/primitivo a la hora de construir gráficas, esto permitirá a los equipos crear gráficas más custom y al mismo tiempo manteniendo consistencia en el mundo charts.. peeero.. esto abre muchas preguntas que de momento no sabemos responder

De momento creemos que lo mejor es que viva en esa librería extended con muchas probabilidades de terminar integrándose en Mística pero antes necesitamos bajar el ejercicio de las gráficas.

**Reply by lauraocana** [reply]
Entonces lo haremos extended, muchas gracias :)
